cucumber/authenticators_config/features/available_authn.feature

```diff
@@ -1,7 +1,19 @@
 Feature: The list of available authentication providers is discoverable
   through the API.
 
-  Background:
+  Scenario: Verify installed authenticators
+    When I retrieve the list of authenticators
```

Comment:
Given a running master
If we have the test in XRAY we need to add it here as well

Reply:
@eladkug You've mismatched repos. There's no master in OSS so there's no any given.

Reply:
Will add a test to XRAY.

Reply:
@sashaCher How are we using XRAY?  Are we duplicating the cucumber tests somewhere?

Reply:
> Are we duplicating the cucumber tests somewhere

Briefly. See [ONYX-12104](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-12104) for example in Jira and in the PR

Reply:
@jonahx for now we using XRAY for manual testing only and the cucumber represent the test.
In the future we plan to update the feature from our repo